index.html

```diff
@@ -31,48 +50,79 @@ <h3>"Experience Professional-grade Sound: Record Your Next Hit with us."</h3>
       </div>
       <div class="visit-time">
       <h2>
-        Visit us any time: MON-SAT 8h00-18h00.
+        Visit us any time from MON-SAT between 8h00-18h00.
       </h2>
```

Comment:
Great design @God-Logic 🤝

- [ ] Kindly ensure your design is responsive to all screen sizes. As you can see in the image below your text tends to overlap as the screen size reduces.
![responsive](https://user-images.githubusercontent.com/63917725/233805646-5f7ab9c6-6f19-4b40-b973-9b67fe32f50f.PNG)